### PR TITLE
[#820]: fix course error (connects #820)

### PR DESCRIPTION
**Pull request by Pr0chin**

### CODE REVIEW

**Review by dogi — APPROVED** [review]

File: app/MyApp/app/views/CourseLevelsTable.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ $(function () {
                 if (sstatus[index].length > 1) {
                     this.vars.lastAttemptStatus = App.languageDict.attributes.Attempted;
                 }
+                console.log(sr[index][totalattempt[index]],(sr[index] instanceof Array))
```
Comment (lmmrssa) [inline app/MyApp/app/views/CourseLevelsTable.js]: can be removed

File: app/MyApp/app/views/CourseStepStatistics.js [hunk 2]
```diff
@@ -47,7 +47,8 @@ $(function() {
                 this.vars.Total_Percentage = totalpercentEachQuestion
                 this.vars.Last_Attempt = attempt
                 this.vars.Total_Question = this.model.attributes.questionslist.length
-                this.vars.StepType = questionlist.attributes.stepType
+                console.log(questionlist.attributes.stepType)
```
Comment (lmmrssa) [inline app/MyApp/app/views/CourseStepStatistics.js]: can be removed

File: app/MyApp/app/views/takeTestView.js [hunk 3]
```diff
@@ -287,6 +288,7 @@ $(function() {
                 }
                 this.answersave(pqattempts[this.stepindex]);
                 sstatus[this.stepindex][pqattemptss[this.stepindex]] = this.sstatus
+                console.log(sstatus[this.stepindex][pqattemptss[this.stepindex]])
```
Comment (lmmrssa) [inline app/MyApp/app/views/takeTestView.js]: can be removed

File: app/MyApp/index.html [hunk 4]
```diff
@@ -1209,7 +1209,9 @@
             <% for(var k = 0; k < Totalfailstat.length; k++) {%>
                 <tr>
                     <td><%=member_Name[k]%></td>
-                    <td><%=Totalfailstat[k]%></td>
+                    <% for(var j = 0; j < steps.length; j++) {%>
```
Comment (lmmrssa) [inline app/MyApp/index.html:1212]: not sure how it looks with the loop

**Review by Rupesh87 — APPROVED** [review]

**Comment by razu9861** [comment]
@Pr0chin are those console necessary

**Comment by razu9861** [comment]
@Pr0chin are those console necessary

**Review by lmmrssa — CHANGES_REQUESTED** [review]

**Review by lmmrssa — APPROVED** [review]

**Pull request closed without merging**